tests/installation/await_install.pm

```
@@ -125,6 +125,13 @@ sub run {
     }
     # on s390 we might need to install additional packages depending on the installation method
     if (check_var('ARCH', 's390x')) {
+      #For poo#93949, we need password possibility
+      if (!is_sle && !is_leap) {
+        select_console 'install-shell';
+        assert_script_run("mkdir -p /mnt/etc/ssh/sshd_config.d");
```

To fail early if the race is lost:

```suggestion
        assert_script_run('mountpoint /mnt && mkdir -p /mnt/etc/ssh/sshd_config.d');
```